align/pnr/checker.py

```diff
@@ -54,7 +54,7 @@ def check_placement(placement_verilog_d, scale_factor):
                         lly=bbox.lly/scale_factor,
                         urx=bbox.urx/scale_factor,
                         ury=bbox.ury/scale_factor,
-                        sx=1,  # TODO: change back to sX
-                        sy=1  # TODO: change back to sY
+                        sx=t['sX'],
```

Comment:
This does not solve the issue.

Reply:
You are right. Here is a failure: `cd tests/pdk/finfet && pytest -v test_circuits.py -k cmp_2 --runnightly`

All blocks are overlapping. @854768750 , any thoughts on why this would be the case?

Reply:
let me see

Reply:
This is because no placement is found at the top level. Then all blocks are at (0, 0). I am looking into this why no placement is found.

Reply:
I looked into this test case. Each block has aspect ratio<0.55. There is also a veritcal ordering constraint of length 5. However, the aspect ratio constraint lower bound is 0.5 and that is why no solution is generated.

Set aspect ratio low bound 0.1 works.

Reply:
@soneryaldiz the solution has aspect ratio 0.2, so lower bound 0.1 would be fine.

Reply:
@854768750 , there is a solution to this test case satisfying the aspect ratio. Please try the same test case on master and you should get the following: `bbox: [0, 0, 10800, 17640]`. Not sure why you think each block has aspect ratio <0.55. For example, diff pair variants span 0.09 to 12.5.

Reply:
@soneryaldiz What is -nvariant in this test case?

Reply:
nvariant is 8 (default value in `tests/pdk/finfet/utils.py::run_example`

Reply:
`pytest -v test_circuits.py -k cmp_2 --runnightly` is correct now.